Builds/macos/README.md

```diff
@@ -0,0 +1,206 @@
+# macos Build Instructions
+
+## Important
+
+We don't recommend macos for rippled production use at this time. Currently, the
+Ubuntu platform has received the highest level of quality assurance and
+testing. That said, macos is suitable for many development/test tasks.
+
+## Prerequisites
+
+You'll need macos 10.8 or later.
+
+To clone the source code repository, create branches for inspection or
+modification, build rippled using clang, and run the system tests you will need
+these software components:
+
+* [XCode](https://developer.apple.com/xcode/)
+* [Homebrew](http://brew.sh/)
+* [Boost](http://boost.org/)
+* other misc utilities and libraries installed via homebrew
+
+## Install Software
+
+### Install XCode
+
+If not already installed on your system, download and install XCode using the
+appstore or by using [this link](https://developer.apple.com/xcode/).
+
+For more info, see "Step 1: Download and Install the Command Line Tools"
+[here](http://www.moncefbelyamani.com/how-to-install-xcode-homebrew-git-rvm-ruby-on-mac)
+
+The command line tools can be installed through the terminal with the command:
+
+```
+xcode-select --install
+```
+
+### Install Homebrew
+
+> "[Homebrew](http://brew.sh/) installs the stuff you need that Apple didn’t."
+
+Open a terminal and type:
+
+```
+ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+```
+
+For more info, see "Step 3: Install Homebrew"
```

Comment:
I think that should read, `see "Step **2**: Install Homebrew".

Reply:
good catch - I fixed the link and the text.